src/gapstate.h

```diff
@@ -117,9 +117,11 @@ typedef struct GAPState {
     Obj ExecState;
 
     /* From opers.c */
+#if defined(HPCGAP)
```

Comment:
I'd be happy if we change all occurrences of `#ifdef HPCGAP` to `#if defined(HPCGAP)` (and similar for any `#ifndef`, though I already tried to avoid them, as they are very easy to mistake for `#ifdef` -- but perhaps this can be done in one big commit, after the current syncing commits have been merged? That would be less work for me right now :-)